misc/logging/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "libp2p-logging"
+version = "0.1.0"
+edition = "2021"
+authors = ["Krishang Shah <shahkrishang11@gmail.com>"]
+license = "MIT"
+repository = "https://github.com/libp2p/rust-libp2p"
+publish = false
```

Comment:
if this is set to false we cannot use it on the rest of the code

Reply:
> if this is set to false we cannot use it on the rest of the code

Right now, it is only used as a `dev-dependency` everywhere other than `protocols/perf` and `misc/server`.
If we only use it as a `dev-dependency`, we don't need to publish it. So I think it is best not to publish it and I can remove the usage of `libp2p-logging` from `protocols/perf` and `misc/server`. wdyt?

Reply:
or am I missing something. I could be wrong though :)

Reply:
yeah but both `protocols/perf` and `misc/server` are published, see [1](https://crates.io/crates/libp2p-perf) and [2](https://crates.io/crates/libp2p-server)

Reply:
> yeah but both `protocols/perf` and `misc/server` are published, see [1](https://crates.io/crates/libp2p-perf) and [2](https://crates.io/crates/libp2p-server)

done